doc/modules/ROOT/pages/nrepl-api/ops.adoc

```diff
@@ -309,7 +309,7 @@ Required parameters::
 {blank}
 
 Optional parameters::
-* `:class` A Java class. If ``:ns`` is passed, it will be used for fully-qualifiying the class, if necessary.
+* `:class` A Java class. If ``:ns`` is passed, it will be used for fully-qualifying the class, if necessary.
```

Comment:
.adoc files auto-generated, please fix the source instead and re-run `lein docs`

Reply:
Thanks. Made the changes.